lib/devices/ios/ios.js

```diff
@@ -778,57 +784,33 @@ IOS.prototype.setSafariPrefs = function () {
   }
 };
 
-IOS.prototype.detectTraceTemplate = function (cb) {
```

Comment:
where did this method go, did it go into xcode.js? noticed you removed it from the main flow in both RD and sim paths